src/cli.rb

```diff
@@ -150,6 +150,19 @@ def run
         c.action Commands::Status
       end
 
+      command :ipmi do |c|
+        c.syntax = 'metal ipmi NODE_IDENTIFIER [options]'
+        c.summary = 'Perform ipmi commands on single or multiple machines'
+        c.description = "***VOLATILE***\n\n" \
+                    'Perform ipmi commands on single or multiple machines'
+        #c.example 'description', 'command example'
+        c.option '-g',
+          'Specifies that NODE_IDENTIFIER is the group. MUST be before NODE_IDENTIFIER'
```

Comment:
Having to specify `-g` before the `NODE_IDENTIFIER` seems inconsistent with our other commands, how complicated do you think it would be to change to allow it anywhere? If this is simple could do this now, otherwise just add a card for it.